src/FINDOLOGIC/Definitions/OrderType.php

```diff
@@ -13,13 +13,13 @@ class OrderType
     const NEWEST_PRODUCTS_FIRST = 'dateadded DESC';
```

Comment:
I am doing this already. I just added the `DESC` property for some cases, where it actually makes sense (expensive products first), but I removed Z-A because that may not be useful for customers.